.github/workflows/wait_for_required_workflows.yml

```diff
@@ -81,7 +81,7 @@ jobs:
               const runs = checkRuns.map(({ name, conclusion }) => ({ name, conclusion }))
               console.log(`Got the following check runs: ${JSON.stringify(runs)}`)
               const matchingRuns = runs.filter(({ name }) => actions.includes(name))
-              const failedRuns = matchingRuns.filter(({ conclusion }) => conclusion !== 'success')
+              const failedRuns = matchingRuns.filter(({ conclusion }) => conclusion !== 'success' && conclusion !== 'skipped')
```

Comment:
Is this change related to the caching change? Is so let's move it to a separate PR so we can test it separately and also revert it easily if it breaks something

Reply:
it is related to skip in `validate-release`, I'll extract it

Reply:
Extracted to https://github.com/cloudquery/cloudquery/pull/9668